nano/core_test/uint256_union.cpp

```diff
@@ -549,7 +549,7 @@ void test_union_operator_less_than ()
 template <typename Num>
 void check_operator_less_than (Num lhs, Num rhs)
 {
-	ASSERT_TRUE (lhs < rhs);
```

Comment:
We should leave the check_operator_ function as they are since they are specifically checking the operator. ASSERT_LT likely uses the operator internally but in theory it doesn't. It could check for less than like this for example: !(a >= b), which wouldn't be testing the less than operator.

Reply:
Ok, I've reverted this